workspace-loader/.devfile

```diff
@@ -0,0 +1,57 @@
+specVersion: 0.0.1
+name: workspace-loader
+projects:
+  - name: che
+    source:
+      type: git
+      location: 'https://github.com/eclipse/che.git'
+components:
+  - alias: theia-editor
+    type: cheEditor
+    id: eclipse/che-theia/next
+  - alias: exec-plugin
+    type: chePlugin
+    id: eclipse/che-machine-exec-plugin/0.0.1
+  - alias: ws-loader-dev
+    type: dockerimage
+    image: 'eclipse/che-dashboard-dev:nightly'
+    endpoints:
+      - name: dev-server
+        port: 3000
+        attributes:
+          path: /
+          protocol: http
+          public: "true"
+    mountSources: true
+    memoryLimit: 1Gi
+commands:
+  - name: '[workspace loader] install dependencies'
+    actions:
+      - type: exec
+        component: ws-loader-dev
+        command: cd /projects/che/workspace-loader && yarn
```

Comment:
Why Do you need `cd` if you defined  `workdir`? And the same for each command...